src/SDK/Client/Platform/methods/documents/get.ts

```diff
@@ -9,12 +11,18 @@ import {Platform} from "../../Platform";
  */
 declare interface fetchOpts {
     where: any;
-    orderBy: any;
-    limit: number;
-    startAt: number;
-    startAfter: number;
+    orderBy?: any;
+    limit?: number;
+    startAt?: number;
+    startAfter?: number;
 }
 
+type WhereCondition = [
+    string,
+    string,
```

Comment:
`'<' | '<=' | '==' | '>' | '>=' | 'in' | 'startsWith' | 'elementMatch' | 'length' | 'contains'`